CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Logger.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CallAutomation_AppointmentReminder
+{
+    //Caution: Logging should be removed/disabled if you want to use this sample in production to avoid exposing sensitive information
+    public static class Logger
```

Comment:
is there a reason we wouldn't use standard ILogger implementation? It logs to the console by default.

Reply:
Sure, I will switch it to ILogger. Thanks